Makefile

```diff
@@ -243,22 +243,67 @@ verify-crd:
 	$(diff) '$(tmp_file)' '$(CRD_PATH)' || (echo 'File $(CRD_PATH) is not up-to date. Please run `make update-crd` to update it.' && false)
 .PHONY: verify-crd
 
-# $1 - target dir
-define generate-dev-manifests
-	mkdir -p '$(1)' && \
-	cp deploy/manager/prod/*.yaml '$(1)' && \
-	yq eval '.spec.cpuset=false | .spec.datacenter.racks[] = (.spec.datacenter.racks[] | .resources.limits.cpu="200m" | .resources.limits.memory="200Mi" | .resources.requests.cpu="10m" | .resources.requests.memory="100Mi" )' -i '$(1)'/10_scylla_scyllacluster.yaml
+# $1 - name
+define generate-namespace
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: $(1)
 endef
 
-update-deploy-dev:
-	$(call generate-dev-manifests,deploy/manager/dev)
-.PHONY: update-deploy-dev
+# $1 - name
+# $2 - chart path
+# $3 - values path
+# $4 - target dir
+define generate-manifests
+	mkdir -p '$(4)'
+	$(file > $(4)/00_namespace.yaml,$(generate-namespace))
```

Comment:
`$(file >` can you elaborate on what this does? I am not familiar with such syntax in the makefile, couldn't find buildin function `file` in make reference either.

Reply:
no longer used, but it allows to save templatized definition in file. https://www.gnu.org/software/make/manual/html_node/File-Function.html